linux/README.md

```diff
@@ -21,12 +20,12 @@ Documentation generation
 To generate a walkthrough file corresponding to a given OS, run the
 `autogenerate.sh` script, the OS is specified as a paramater e.g.
```

Comment:
not this PR's fault but maybe while here fix "parameter" too

Reply:
done